src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -1357,10 +1359,59 @@ public RecipeMap<?> getRecipeMap() {
         return null;
     }
 
+    public void checkWeatherOrTerrainExplosion(float explosionPower, double additionalFireChance, IEnergyContainer energyContainer) {
```

Comment:
I think it would be good to check the config here as well to avoid an accidental call and exit early